### ISSUE TITLE
Some podcasts don't work

### ISSUE THREAD

**Issue by rconnorjohnstone** [Rigellute/spotify-tui#671]
I'm not really sure what is wrong with these specific podcasts, but while most podcasts work just fine by searching for them, certain podcasts give an error.

For instance, I can search for "reply all" and pull up the reply all podcast just fine. But if I search for "enormocast" and pull that up (it should be the only search result if you'd like to recreate yourself) I get this:

![image](https://user-images.githubusercontent.com/45497336/101210488-cc5f6280-3632-11eb-8246-04cb61d1dd7e.png)

It looks like something to do with the JSON data for certain podcasts isn't converting properly.

**Comment by slabach** [comment]
yeah, I'm actually finding this more often than not with podcasts. Very few actually seem to work for me. Came here looking for a solution
